.githooks/pre-commit

```diff
@@ -6,11 +6,12 @@ git apply -R unstaged.diff
 
```

Comment:
This isn't related to the upgrade, but I needed it to be able to commit the files 😛